src/packages/@ncigdc/modern_components/AffectedCasesTable/AffectedCasesTable.js

```diff
@@ -362,11 +362,13 @@ const Component = compose(
                     gender: c.demographic ? c.demographic.gender : '',
                     age_at_diagnosis: ageDisplay(diagnosis.age_at_diagnosis),
                     tumor_stage: diagnosis.tumor_stage,
-                    days_to_last_follow_up: (diagnosis.days_to_last_follow_up ||
-                      0)
-                      .toLocaleString(),
-                    days_to_death: (diagnosis.days_to_death || 0)
-                      .toLocaleString(),
+                    days_to_last_follow_up: (diagnosis.days_to_last_follow_up !=
```

Comment:
personally I just think it makes the code harder to read. Now I have to figure out if thats on purpose or not :D
I would be in favour of adding the `eqeqeq` rule back :D